test/connection.test.js

```diff
@@ -1547,4 +1547,12 @@ describe('connections:', function() {
       // currently cannot write test for continueOnError or errors in general.
     });
   });
+  describe('processConnectionOptions', function() {
```

Comment:
Add an `after()` hook to this `describe()` that calls `await m.disconnect()`. Right now this test leaves a dangling connection.